frontend/src/feature/projects/Projects.tsx

```diff
@@ -1,123 +1,104 @@
-      onDelete();
-    }
-    handleClose();
-  }
+const getItemStyle = (isDragging: boolean, draggableStyle: any) => ({
+  // some basic styles to make the items look a bit nicer
+  userSelect: "none",
+  padding: grid * 2,
+  margin: `0 0 ${grid}px 0`,
 
-  function handleEditProject() {
-    history.push(`${url}/edit/${project.id}`);
-    handleClose();
-  }
+  // change background colour if dragging
+  background: isDragging ? "lightgreen" : "grey",
 
-  function handleClose() {
-    setAnchorEl(null);
-  }
-
-  function handleClick(event: any) {
-    setAnchorEl(event.currentTarget);
-  }
-}
+  // styles we need to apply on draggables
+  ...draggableStyle,
+});
+const getListStyle = (isDraggingOver: boolean) => ({
+  background: isDraggingOver ? "lightblue" : "lightgrey",
+  padding: grid,
+  width: 250,
+  minHeight: 250,
+});
 
 export function Projects() {
-  const [state, setState] = useState<Project[]>([]);
+  const [state, setState] = useState<Project[][]>(STATES.map((s) => []));
   const { path, url } = useRouteMatch();
   const [trigger, setTrigger] = useState(true);
 
   useEffect(() => {
     getProjects().then((projects) => {
-      setState(projects);
+      const projectLanes = STATES.map((state) =>
+        projects.filter((project) => project.state === state)
+      );
+      setState(projectLanes);
     });
   }, [trigger]);
 
   return (
     <>
       <Switch>
         <Route exact path={path}>
-          {state.map((project) => (
-            <ProjectCard
-              key={project.id}
-              project={project}
-              onDelete={() => setTrigger(!trigger)}
-            />
-          ))}
           <Button variant="contained" color="primary" href={`${url}/new`}>
             New project
           </Button>
+          <DragDropContext onDragEnd={onDragEnd}>
+            <Grid container spacing={3}>
+              {STATES.map((status, ind) => (
+                <Droppable key={ind} droppableId={status}>
+                  {(provided, snapshot) => (
+                    <Grid item xs={3}>
+                      <Typography variant="h4" component="h2">
+                        {status.replace("_", " ")}
+                      </Typography>
+                      <Card
+                        ref={provided.innerRef}
+                        style={getListStyle(snapshot.isDraggingOver)}
+                      >
+                        {state[ind].map((project, index) => (
+                          <Draggable
+                            key={project.id}
+                            draggableId={project.id}
+                            index={index}
+                          >
+                            {(provided, snapshot) => (
+                              <div
+                                ref={provided.innerRef}
+                                {...provided.draggableProps}
+                                {...provided.dragHandleProps}
+                                style={getItemStyle(
```

Comment:
Possible to use useStyles with className? 

const useStyles = makeStyles(theme => ({
  dragging: {
    color: "WHITE"
  },
 notDragging: {
    color: "BLLACK"
  },
}))

const classes = useStyles()

  className={snapshot.isDragging ? classes.dragging : classes.noDragging}